frontend/src/pages/Game/Quizz/Question/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState, useEffect } from "react";
+import dancers from "../../../../utils/dancers";
+import "./quizz.css";
+
+interface IQuestion {
+  track: string;
+  choices: string[];
+  incrementScore: () => void;
+  incrementIndex: () => void;
+}
+
+const Question: React.FC<IQuestion> = ({
+  track,
+  choices,
+  incrementScore,
+  incrementIndex
+}: IQuestion) => {
+  const [dancer] = useState<string>(
+    dancers[Math.floor(Math.random() * dancers.length)]
+  );
+  const [remainingTime, setremainingTime] = useState<number>(30);
+  const [isNextTrackAvailable, setIsNextTrackAvailable] = useState<boolean>(
+    false
+  );
+
+  useEffect(() => {
+    setTimeout(() => {
+      if (remainingTime > 0 && isNextTrackAvailable === false) {
+        setremainingTime(remainingTime - 1);
+      }
+
+      if (remainingTime === 0) {
+        handleTimeEnd();
+      }
+    }, 1000);
+  }, [remainingTime]);
+
+  const handleAnswer = () => () => {
+    handleTimeEnd();
+    // TODO : Request to have the rigth answer ?
+    const isAnswerTrue = true;
+
+    if (isAnswerTrue) {
+      incrementScore();
+    }
+  };
+
+  const handleTimeEnd = (): void => {
+    (document.getElementById("player") as HTMLAudioElement).pause();
```

Comment:
Should we use the React hook UseRef to reference the html element "audio" instead of using "getElementById" ?
-> See [UseRef](https://fr.reactjs.org/docs/hooks-reference.html#useref)

Reply:
Could be q good idea